src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -46,8 +44,8 @@ type PropsType = {|
   extras: ?Array<ExtraModel>,
   postFeedbackDataOverride?: FeedbackParamsType => void,
   closeFeedbackModal: () => void,
-  feedbackSent: SendingStatusType,
-  onSubmit: (feedbackSent?: SendingStatusType) => void,
+  sendingStatus: SendingStatusType,
+  onSubmit: (sendingStatus?: SendingStatusType) => void,
```

Comment:
```suggestion
  onSubmit: (sendingStatus: SendingStatusType) => void,
```
No need for an optional type.